docs/libcurl/curl_getdate.3

```diff
@@ -110,5 +109,5 @@ On systems with 64 bit time_t: if the year is less than 1583, this function
 will return -1. (The Gregorian calendar was first introduced 1582 so no "real"
 dates in this way of doing dates existed before then.)
 .SH "SEE ALSO"
-.BR curl_easy_escape "(3), " curl_easy_unescape "(3), "
-.BR CURLOPT_TIMECONDITION "(3), " CURLOPT_TIMEVALUE "(3) "
+.BR curl_easy_escape "(3), " curl_easy_unescape "(3),"
+.BR CURLOPT_TIMECONDITION "(3), " CURLOPT_TIMEVALUE "(3)"
```

Comment:
This one puzzles me.
1. `mandoc -Tlint curl_getdate.3` on my debian unstable does not warn on this for me
2. I count a total of **79** similar lines, only counting `.3` files in the same directory. Shouldn't then all of them be fixed at the same time?

Reply:
This one is not from mandoc, but from test-groff, which is not something I have installed but it makes sense I guess, the finding (from the submitter of the patch) was:
```
From "test-groff -b -mandoc -T utf8 -rF0 -t -w w -z":

[ "test-groff" is a developmental version of "groff" ]

troff: <curl_getdate.3>:108: warning: trailing space
troff: <curl_getdate.3>:109: warning: trailing space
```

There's a good chance it can be ignored if you think it's fine (it might be too cosmetic to spend the time to fix across all the files).

Reply:
It feels very arbitrary to only fix it in a single file. If we consider this a problem then we should fix it everywhere. If it isn't a problem, then we don't need to fix any file. Why is it a problem? I don't spot any difference at all in the generated result I get with `man -l curl_getdate.3`.

Reply:
Let's drop it then.

I believe this means there's only a single legitimate change to the whole CR, which is for:
`mandoc: curl_getdate.3:64:2: WARNING: skipping paragraph macro: PP empty`